src/audio/mux/mux_generic.c

```diff
@@ -525,8 +525,8 @@ void demux_prepare_look_up_table(struct comp_dev *dev)
 			for (k = 0; k < PLATFORM_MAX_CHANNELS; k++) {
 				if (cd->config.streams[i].mask[j] & BIT(k)) {
 					/* MUX component has only one sink */
```

Comment:
@bkokoszx Please fix the comment while you're at it

Reply:
Updated